deps/Makefile

```diff
@@ -78,3 +78,13 @@ memkind: .make-prerequisites
 	./build-memkind.sh
 
 .PHONY: memkind
+
+jemalloc: .make-prerequisites
+	@printf '%b %b\n' $(MAKECOLOR)MAKE$(ENDCOLOR) $(BINCOLOR)$@$(ENDCOLOR)
+#	cd jemalloc_JE/ && ./autogen.sh
+#	mkdir -p jemalloc_JE/obj
+	cd jemalloc_JE && ./autogen.sh
+	cd jemalloc_JE && ./configure --with-jemalloc-prefix=je_ --with-private-namespace= --without-_export
```

Comment:
Please correct to "--without-export" if that was your intension ( causes build to fail).